### ISSUE TITLE
[BUG]: Muramasa Rebirth [PCSB00404] randomly crashes if NGS is enabled (otherwise there is no audio and no crash)

### ISSUE THREAD

**Issue by v-fox** [Vita3K/Vita3K#3256]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

20240319T211715.cb15157c

### Description of Issue

When starting the game it will crash in first few screen randomly. Music may or may not cut out randomly too some time before the crash.

### Expected Behavior

Not crash.

### Reproduction Steps

Start the game.

### Log File

```gdb
[16:41:12.532] |E| [send]: Decode failure with code -2113929214

Thread 45 "CRI Atom AudioO" received signal SIGSEGV, Segmentation fault.
[Switching to Thread 0x7fff5e6006c0 (LWP 25683)]
0x00005555563a0d0f in ngs::Atrac9Module::process (this=0x7fff802720e0, kern=..., mem=..., thread_id=72, data=..., scheduler_lock=..., voice_lock=...)
    at /home/abuild/rpmbuild/BUILD/vita3k-20240319T211715.cb15157c/vita3k/ngs/src/modules/atrac9.cpp:328
...
#0  0x00005555563a0d0f in ngs::Atrac9Module::process (this=0x7fff802720e0, kern=..., mem=..., thread_id=72, data=..., scheduler_lock=..., voice_lock=...)
    at /home/abuild/rpmbuild/BUILD/vita3k-20240319T211715.cb15157c/vita3k/ngs/src/modules/atrac9.cpp:328
#1  0x00005555563a1650 in ngs::VoiceScheduler::update (this=<optimized out>, kern=..., mem=..., thread_id=<optimized out>)
    at /home/abuild/rpmbuild/BUILD/vita3k-20240319T211715.cb15157c/vita3k/ngs/src/scheduler.cpp:142
#2  0x00005555561383a9 in export_sceNgsSystemUpdate (emuenv=..., thread_id=thread_id@entry=72, export_name=<optimized out>, system=<optimized out>)
    at /home/abuild/rpmbuild/BUILD/vita3k-20240319T211715.cb15157c/vita3k/modules/SceNgsUser/SceNgs.cpp:431
#3  0x000055555614ae51 in call<unsigned int, ngs::System*, 0ul> (export_fn=0x555556138180 <export_sceNgsSystemUpdate(EmuEnvState&, int, char const*, ngs::System*)>, 
    export_name=0x5555556c7460 "sceNgsSystemUpdate", args_layout=..., state=..., thread_id=72, cpu=..., emuenv=...)
    at /home/abuild/rpmbuild/BUILD/vita3k-20240319T211715.cb15157c/vita3k/module/include/module/bridge.h:39
#4  bridge<unsigned int, ngs::System*>(unsigned int (*)(EmuEnvState&, int, char const*, ngs::System*), char const*)::{lambda(EmuEnvState&, CPUState&, int)#1}::operator()(EmuEnvState&, CPUState&, int) const (
    this=0x7fff28038320, emuenv=..., cpu=..., thread_id=72) at /home/abuild/rpmbuild/BUILD/vita3k-20240319T211715.cb15157c/vita3k/module/include/module/bridge.h:60
#5  0x0000555555d75995 in std::function<void (EmuEnvState&, CPUState&, int)>::operator()(EmuEnvState&, CPUState&, int) const (this=0x7fff5e5fbd38, __args=72, __args=72, __args=72)
    at /usr/bin/../lib64/gcc/x86_64-suse-linux/13/../../../../include/c++/13/bits/std_function.h:591
#6  call_import (emuenv=..., cpu=..., nid=1750009868, thread_id=72) at /home/abuild/rpmbuild/BUILD/vita3k-20240319T211715.cb15157c/vita3k/modules/module_parent.cpp:123
#7  0x00005555563bb47d in std::function<void (CPUState&, unsigned int, int)>::operator()(CPUState&, unsigned int, int) const (this=0x7fff280cc590, __args=72, __args=72, __args=72)
    at /usr/bin/../lib64/gcc/x86_64-suse-linux/13/../../../../include/c++/13/bits/std_function.h:591
#8  CPUProtocol::call_svc (this=0x55555818ed20, cpu=..., svc=<optimized out>, pc=<optimized out>, thread=...) at /home/abuild/rpmbuild/BUILD/vita3k-20240319T211715.cb15157c/vita3k/kernel/src/cpu_protocol.cpp:54
#9  0x00005555563b1775 in ThreadState::run_loop (this=0x7fff80310ca0) at /home/abuild/rpmbuild/BUILD/vita3k-20240319T211715.cb15157c/vita3k/kernel/src/thread.cpp:254
#10 0x00005555563af5ff in thread_function (data=<optimized out>) at /home/abuild/rpmbuild/BUILD/vita3k-20240319T211715.cb15157c/vita3k/kernel/src/kernel.cpp:74
#11 0x00007ffff775cddf in ?? () from /lib64/libSDL2-2.0.so.0
#12 0x00007fffefe92bb2 in start_thread () from /lib64/libc.so.6
#13 0x00007fffeff1400c in clone3 () from /lib64/libc.so.6
```

### System Configuration

CPU: Intel Xeon E5-2697v2
GPU/Driver: AMD Radeon RX580 / Mesa-24.0.3
RAM: 4x4Gb ECC DDR3
OS: openSUSE Tumbleweed GNU/Linux

**Comment by Macdu** [comment]
Game related issues must be reported in the compatibility repository (and additionally this issue along with a fix are already described there)

**Comment by v-fox** [comment]
> Game related issues must be reported in the compatibility repository (and additionally this issue along with a fix are already described there)

Didn't know "the compatibility repository" existed and emulators crashes had to go there.
Hard crashes are not usually considered 'compatibility issues'. And if there is "a fix" then why it's not already incorporated into the app?
Care to link it here?

I've searched manually and the only relevant suggestion I saw was to "update the game to v1.06" but pressing "update" also crashes with [gdb.txt](https://github.com/Vita3K/Vita3K/files/14716906/gdb.txt)
Even if it worked it would still be a workaround held on prayers unless original hardware crashes whole firmware too.
